com.microsoft.mrtk.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -164,6 +164,20 @@ public override void ProcessInteractor(XRInteractionUpdateOrder.UpdatePhase upda
                     {
                         attachTransform.rotation = PlayspaceUtilities.ReferenceTransform.rotation * device.deviceRotation.ReadValue();
                     }
+                    else
+                    {
+                        // If we don't have a valid device pose, let's use the palm pose; closest thing we've got!
+                        if (handsAggregator == null)
+                        {
+                            handsAggregator = HandsUtils.GetSubsystem();
+                        }
+                        if (handsAggregator != null &&
+                            xrController is ArticulatedHandController handController &&
+                            handsAggregator.TryGetJoint(TrackedHandJoint.Palm, handController.HandNode, out HandJointPose palmPose))
+                        {
+                            attachTransform.rotation = palmPose.Rotation;
```

Comment:
Same question about applying `PlayspaceUtilities.ReferenceTransform.rotation *`

Reply:
Thanks so much for this catch, would've regretted this later

Reply:
Done